usr/local/www/guiconfig.inc

```diff
@@ -266,11 +266,15 @@ function do_input_validation($postdata, $reqdfields, $reqdfieldsn, &$input_error
 }
 
 function print_input_errors($input_errors) {
+    print_errors($input_errors, gettext('The following input errors were detected:'));
```

Comment:
You are outputting 1 error about 1 host on 1 page. How does this warrant a generic method? Looking at the code a lot of people like doing it that way @ pfsense but I don't like it. You create a generic function after you have more then 1 usecase for it. Only then can you know what your parameters will be.

For now, I'd simply output html in diag_dns; something like ```<div class="alert alert-warning" role="alert"><?=htmlspecialchars($host)?> could not be resolved</div>```